doc/source/_static/css/custom.css

```diff
@@ -919,7 +911,7 @@ footer.col.footer > p {
 #kapa-widget-container figure {
     padding: 0 !important;
   }
-  
+
   .mantine-Modal-root figure {
     padding: 0 !important;
-  }
```

Comment:
add new line

Reply:
FYI this will be automatically checked for if/when we merge https://github.com/ray-project/ray/pull/41500, deprecate `format.sh`, and enforce pre-commit hooks as part of a CI job.